qa/pull-tester/rpc-tests.py

```diff
@@ -146,6 +146,7 @@
     'wallet_zero_value.py',
     'threeofthreerestore.py',
     'show_help.py',
+    'errors.py'
```

Comment:
Always add a trailing comma with Python lists, so we don't have to come back and edit this line later.
```suggestion
    'errors.py',
```